src/main/java/com/management/HumanResources/model/TimeOff.java

```diff
@@ -4,5 +4,6 @@
 
 @Data
 public class TimeOff {
+    private int id;
```

Comment:
You used `long` for the `id` in `FirebaseDao.java` on line 45. Shouldn't they be of the same type?